### DISCUSSION TITLE
Unable to remove food

### DISCUSSION THREAD

**Discussion by pippo73** [TandoorRecipes/recipes#2638] [General]
Hi, 
I'm trying to put some order on my site

While I try to remove a food called "80" which in the list looks not connected to any recipe, I get the following error message on the web page:

```
L'elemento che stai cercando di eliminare è ancora in uso e non può essere eliminato.
{"error":true,"msg":"80 è parte dello step di una ricetta e non può essere eliminato"}
```
and: 

```
[2023-09-21 21:20:32 +0200] [16] [DEBUG] DELETE /api/food/2134/
Forbidden: /api/food/2134/
Forbidden: /api/food/2134/
172.18.0.4 - - [21/Sep/2023:21:20:32 +0200] "DELETE /api/food/2134/ HTTP/1.0" 403 88 "https://ricette.leomutti.casa/list/food/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/117.0"
[2023-09-21 21:20:32 +0200] [16] [DEBUG] Closing connection.
```

In the docker log.

How to understand to which recipe is connected?

tnx

**Comment by smilerz** [comment]
steps and ingredients can be orphaned.  go to admin/ingredients and admin/steps and run 'delete steps/ingredients not part of a recipe'
